### PR TITLE
git-lfs 2.6.0

### PR DESCRIPTION
**Pull request by ttaylorr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/git-lfs.rb [hunk 1]
```diff
@@ -27,6 +26,7 @@ def install
       ENV["GEM_HOME"] = ".gem_home"
       system "gem", "install", "ronn"
 
+      system "make", "vendor", "RM=true"
```
Comment (ttaylorr) [inline Formula/git-lfs.rb]: In Git LFS, we remove some of the dependencies that Go takes in by default. In this context, though, they are required. To work around an error on our part, we reinstall the dependencies, and set the `rm` command to instead run `true`, making the "deletions" a no-op. We'll fix this for v2.6.1 which will obviate the need for this section of the diff.

Comment (Moisan) [reply Formula/git-lfs.rb]: Could you add a short comment above this line to explain the situation and to remove this line after v2.6.1?

Comment (SMillerDev) [reply Formula/git-lfs.rb]: @ttaylorr bump

Comment (ttaylorr) [reply Formula/git-lfs.rb]: Sure, and sorry for the late response -- I was out of the office last week and am just catching up on older messages. I resent this as https://github.com/Homebrew/homebrew-core/pull/33619/commits/69dc151cccff16a82ee3dc8d1bed70301f730d94.

**Comment by Moisan** [comment]
Thank you @ttaylorr !

**Pull request closed without merging**